src/karma.conf.js

```diff
@@ -4,20 +4,20 @@
 module.exports = function (config) {
   config.set({
     basePath: "",
-    frameworks: ['jasmine', '@angular-devkit/build-angular'],
+    frameworks: ["jasmine", "@angular-devkit/build-angular"],
```

Comment:
Our coding style in the whole project uses single quotes, codacy should use single quotes instead of us changing to double quotes

Reply:
So shall I make codacy ignore these?

Reply:
Okay I am making Codacy to to use single quotes check. Closing this patch :+1: